### PR TITLE
refactor: utils seperation

### PR DESCRIPTION
**Pull request by KommentatorForAll**
moved db things to their own file and fixed some warnings (there were about 90)

### CODE REVIEW

File: src/helper/startEval.js [hunk 1]
```diff
@@ -2,22 +2,21 @@ import repl from 'repl';
 // eslint-disable-next-line import/no-unresolved
 import { setTimeout } from 'timers/promises';
 import config from '../config';
-import * as util from '../util';
+import { getClient } from '../util';
 
 const defaultContext = {
     config,
-    util,
```
Comment (Chicken) [inline src/helper/startEval.js:9]: Why?

File: src/constants/Blacklist.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ const swearRegex = new RegExp(
             str +
             swear
                 // This function makes the swear allow spaces inbetween letters
-                .replace(/(?:(?:\(.+?\)(?:\{.+?\})?)|[a-z=.-])(?:\+|\?|\*|\$)?/g, '$&\\s*') +
+                .replace(/(?:(\(.+?\)(?:{.+?})?)|[a-z=.-])([+?*$])?/g, '$&\\s*') +
```
Comment (Chicken) [inline src/constants/Blacklist.js:54]: Why?

File: src/modules/errorHandling/unhandledError.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
     console.error(err);
     getClient()?.destroy();
-    userDb.close();
-    responseDb.close();
-    keyDb.close();
-    factionDb.close();
-    process.exit(1);
+    Promise.all([userDb.close(), responseDb.close(), keyDb.close(), factionDb.close()]).then(
+        process.exit(1)
+    );
```
Comment (Chicken) [inline src/modules/errorHandling/unhandledError.js:10]: These are not promises

**Review by Chicken — CHANGES_REQUESTED** [review]

File: src/helper/startEval.js (hunk 1 above)
Comment (KommentatorForAll) [reply src/helper/startEval.js:9]: done automaticly by either prettier or during refactoring

**Review by Chicken — APPROVED** [review]

File: src/constants/Blacklist.js (hunk 2 above)
Comment (WilsontheWolf) [inline src/constants/Blacklist.js:54]: Why are we removing the `?:` here?

File: src/helper/main.js [hunk 4]
```diff
@@ -94,7 +95,7 @@ import FactionDto from '../dto/FactionDto';
             try {
                 console.log('Reading info...');
                 const info = JSON.parse(await fs.readFile('./data/firebase.json'));
-                const ownerOrKeyRegex = /^(?:\w+-\w+-\w+)|(?:owner)$/;
+                const ownerOrKeyRegex = /^(\w+-\w+-\w+)|(owner)$/;
```
Comment (WilsontheWolf) [inline src/helper/main.js:98]: Same here

File: src/modules/modCommands/processCommands.js [hunk 5]
```diff
@@ -50,14 +50,9 @@ async function processCommandsNewMessage(client, message) {
...
-            message.channel
-                .send(`Something went wrong processing that command! Please try again later.`)
-                .catch((err) => console.error('Error sending error!\n', err));
-        else
-            message
-                .reply(`Something went wrong processing that command! Please try again later.`)
-                .catch((err) => console.error('Error sending error!\n', err));
+        message.channel
+            .send(`Something went wrong processing that command! Please try again later.`)
+            .catch((err) => console.error('Error sending error!\n', err));
```
Comment (WilsontheWolf) [inline src/modules/modCommands/processCommands.js:55]: Unnecessary but ok.

**Review by WilsontheWolf — CHANGES_REQUESTED** [review]

File: src/constants/Blacklist.js (hunk 2 above)
Comment (KommentatorForAll) [reply src/constants/Blacklist.js:54]: bc it doesn't do anything. it is just redundant

File: src/modules/modCommands/processCommands.js (hunk 5 above)
Comment (KommentatorForAll) [reply src/modules/modCommands/processCommands.js:55]: the one before was unnessesary code dublication

File: src/constants/Blacklist.js (hunk 2 above)
Comment (WilsontheWolf) [reply src/constants/Blacklist.js:54]: It tells regex not to store this which actually is more performant.

File: src/modules/modCommands/processCommands.js (hunk 5 above)
Comment (WilsontheWolf) [reply src/modules/modCommands/processCommands.js:55]: Well it was replying or sending

Comment (KommentatorForAll) [reply src/modules/modCommands/processCommands.js:55]: does it really matter if it sends or replies? and this is a lot smaller

File: src/constants/Blacklist.js (hunk 2 above)
Comment (KommentatorForAll) [reply src/constants/Blacklist.js:54]: as if such a small change really matters performance wise. if you want to talk about performance, look at the rest of the code

File: src/modules/modCommands/processCommands.js (hunk 5 above)
Comment (WilsontheWolf) [reply src/modules/modCommands/processCommands.js:55]: I guess not

File: src/constants/Blacklist.js (hunk 2 above)
Comment (WilsontheWolf) [reply src/constants/Blacklist.js:54]: Ok fine

File: src/modules/errorHandling/unhandledError.js (hunk 3 above)
Comment (Chicken) [reply src/modules/errorHandling/unhandledError.js:10]: In fact we no longer need to call these at all when we update to latest Enmap

Comment (WilsontheWolf) [reply src/modules/errorHandling/unhandledError.js:10]: This is still an issue

**Review by Chicken — CHANGES_REQUESTED** [review]
Remove db closing entirely

**Review by Chicken — APPROVED** [review]

**Review by WilsontheWolf — APPROVED** [review]

**Pull request merged**